src/builder/builder.go

```diff
@@ -498,8 +498,15 @@ func (b *Builder) AddRPMList(rpms []os.FileInfo) {
 			fmt.Println("ERROR: RPM is not valid! Please make sure it was built correctly.")
 			os.Exit(1)
 		} else {
-			fmt.Printf("Copying %s\n", rpm.Name())
-			helpers.CopyFile(b.Repodir+"/"+rpm.Name(), b.Rpmdir+"/"+rpm.Name())
+			if _, err = os.Stat(b.Repodir + "/" + rpm.Name()); err == nil {
+				continue
+			}
```

Comment:
If err != nil, I suppose the code assumes that you hit an ENOENT condition and can safely try to hardlink. Can you recover from other errors that may occur?